enterprise_extensions/empirical_distr.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
-from __future__ import (absolute_import, division,
-                        print_function)
+
 import logging
+
 import numpy as np
 
 try:
```

Comment:
after axing `Py2.x` we don't need the `cPickle` v. `pickle` checks

Reply:
This thinks I'm talking about the added whitespace, but its the unaltered try/except at line 7.

Reply:
Good call!